Homework/src/main/java/io/codelex/flowcontrol/practice/TenBillion.java

```diff
@@ -0,0 +1,48 @@
+package io.codelex.flowcontrol.practice;
+
+import java.util.Scanner;
+
+public class TenBillion {
+
+    //TODO: Write a Java program that reads an positive integer and count the number of digits the number (less than ten billion) has.
+    public static void main(String[] args) {
+        Scanner in = new Scanner(System.in);
+        System.out.print("Input an integer number less than ten billion: ");
+
+        long n = in.nextLong();
+
+
+        //if (n!=0) {  Not sure if this is even needed because if something other than a number is inserted, the program would stop.
```

Comment:
to var notestēt piemēŗam ar `.hasNextLong()` vai arī vnk nočekot vai cipars ir long range